### PR TITLE
binlog: fix example

### PR DESCRIPTION
**Pull request by GregoryIan**

### CODE REVIEW

**Comment by GregoryIan** [comment]
@kangxiaoning PTAL

File: tools/tidb-binlog-kafka.md [hunk 1]
```diff
@@ -356,7 +356,7 @@ cd pb_reader-latest-linux-amd64
 PbReader 使用示例
 
 ```bash
-./bin/pbReader -binlog-file=binlog-0000000000000000
+./bin/pbReader -binlog-file=path/to/binlog-0000000000000000
```
Comment (liubo0127) [inline tools/tidb-binlog-kafka.md]: 改成这样吧 @kangxiaoning 
./bin/pbReader -binlog-file=${path}/binlog-0000000000000000

Comment (lilin90) [reply tools/tidb-binlog-kafka.md]: @GregoryIan PTAL

**Comment by lilin90** [comment]
@liubo0127 PTAL

**Pull request merged**